Library/Formula/makemake.rb

```diff
@@ -0,0 +1,15 @@
+class Makemake < Formula
+  homepage "https://github.com/Julow/makemake"
+  # desc "Cool Makefile generator (C/C++)"
```

Comment:
`desc` is part of the DSL, like `homepage` and `url`. The formula should look like this:

```ruby
class Makemake < Formula
  desc "the description"
  homepage "https://github.com/Julow/makemake"
  url "..."
  # ...
```